### PR TITLE
Merge wcs recovery to Master

### PR DESCRIPTION
**Pull request by Xen0Xys**

### CODE REVIEW

File: src/ipyaladin/aladin.py [hunk 1]
```diff
@@ -183,6 +187,39 @@ def target(self, target: Union[str, SkyCoord]) -> None:
...
+            An `~astropy.wcs.WCS` object representing the WCS of the widget.
+
+        """
+        if self._wcs == {}:
+            raise ValueError(
+                "You need to call synchronize_wcs before accessing the wcs attribute. "
+                "The WCS attribute need to be accessed from the next cell."
+            )
+        wcs = WCS(naxis=self._wcs["NAXIS"])
+        wcs.wcs.cdelt = [self._wcs["CDELT1"], self._wcs["CDELT2"]]
```
Comment (tboch) [inline src/ipyaladin/aladin.py:209]: can't we just loop through the dictionary keys/values retrieved from aladin.getViewWCS() ?
Would be more generic and less error prone.

Comment (ManonMarchand) [reply src/ipyaladin/aladin.py:209]: Looks like you can do

```python
from astropy.wcs import WCS
WCS({"CRVAL1": 0, "CRVAL2":1})
```

```
WCS Keywords

Number of WCS axes: 2
CTYPE : '' '' 
CRVAL : 0.0 1.0 
CRPIX : 0.0 0.0 
PC1_1 PC1_2  : 1.0 0.0 
PC2_1 PC2_2  : 0.0 1.0 
CDELT : 1.0 1.0 
NAXIS : 0  0
```

Comment (tboch) [reply src/ipyaladin/aladin.py:209]: oh yes, thanks, that's better!

File: src/ipyaladin/aladin.py [hunk 2]
```diff
@@ -205,15 +205,9 @@ def wcs(self) -> WCS:
...
-        wcs.wcs.cdelt = [self._wcs["CDELT1"], self._wcs["CDELT2"]]
-        wcs.wcs.crpix = [self._wcs["CRPIX1"], self._wcs["CRPIX2"]]
-        wcs.wcs.crval = [self._wcs["CRVAL1"], self._wcs["CRVAL2"]]
-        wcs.wcs.ctype = [self._wcs["CTYPE1"], self._wcs["CTYPE2"]]
-        wcs.wcs.cunit = [self._wcs["CUNIT1"].strip(), self._wcs["CUNIT2"].strip()]
-        wcs.wcs.radesys = self._wcs["RADECSYS"].strip()
-        wcs.array_shape = (self._wcs["NAXIS2"], self._wcs["NAXIS1"])
-        return wcs
+        if "RADECSYS" in self._wcs:  # RADECSYS keyword is deprecated for astropy.WCS
+            self._wcs["RADESYS"] = self._wcs.pop("RADECSYS")
```
Comment (ManonMarchand) [inline src/ipyaladin/aladin.py:209]: Indeed... Looks like this got deprecated in the [fits standard 4.0](https://fits.gsfc.nasa.gov/standard40/fits_standard40aa-le.pdf) 
We should propagate upstream in AL

**Pull request closed without merging**